ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/KsqlRestApplicationTest.java

```diff
@@ -140,93 +124,6 @@ public void shouldCloseServiceContextOnClose() {
     verify(serviceContext).close();
   }
 
-  @Test
-  public void shouldCreateCommandTopicIfItDoesNotExist() {
```

Comment:
None of these test cases actually verify that the rest app creates the topic. Happy to add one, but I'd prefer to do it in another PR

Reply:
Ah yes, they're all just testing a method that shouldn't of been accessible anyway.

TBH, given you're changing the code in this class that calls out to create the topic, I think it reasonable that this PR adds a test to make sure the call is made.

Reply:
I'm all for covering the testing gap. I'd really rather do it from another PR (that I'll do right away I promise) though since to do it nicely involves some refactoring of KsqlRestApplication.

Reply:
Cool.